django/bossspatialdb/views.py

```diff
@@ -488,3 +488,117 @@ def delete(self, request, collection, experiment, channel):
         channel_obj.save()
 
         return HttpResponse(status=204)
+
+class CutoutToBlack(APIView):
+    """
+    View to handle spatial cutouts by providing all datamodel fields
+
+    * Requires authentication.
+    """
+    # Set Parser and Renderer
+    parser_classes = (BloscParser, BloscPythonParser, NpygzParser, BrowsableAPIRenderer)
+    renderer_classes = (BloscRenderer, BloscPythonRenderer, NpygzRenderer, JpegRenderer,
+                        JSONRenderer, BrowsableAPIRenderer)
+
+    def __init__(self):
+        super().__init__()
+        self.data_type = None
+        self.bit_depth = None
+    
+    def put(self, request, collection, experiment, channel, resolution, x_range, y_range, z_range, t_range=None):
+        """
+        View to handle PUT requests for a overwriting a cuboid to 0s 
+
+        Due to parser implementation, request.data should be a numpy array already.
```

Comment:
This may confuse someone as there is no data being given to this API, so request.data shouldn't contain anything